.changeset/fix-tag-complexity-detection.md

```
@@ -0,0 +1,7 @@
+---
+"task-master-ai": patch
+---
+
+Fix expand command to properly detect tag-specific complexity reports
```

markdownlint-cli2 (0.17.2)

First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)